### PR TITLE
Api role enabled for read only routers #37

### PR DESCRIPTION
**Pull request by pnvasanth**
Hi @akudiyar 
  
Actually #37  the statistics are stored in storage notes under the space name called `_queue_statistics`.  which is [initializing](https://github.com/tarantool/sharded-queue/blob/master/sharded_queue/storage.lua#L90) only in storage master nodes.  

Router nodes are  [collecting](https://github.com/tarantool/sharded-queue/blob/master/sharded_queue/api.lua#L407) the statistics details from all the storage masters and merging it before return.
So we are good to enable the api roles in all the router nodes. 

Thank you

### CODE REVIEW

**Comment by akudiyar** [comment]
Thank you for the patch! I have no questions about the code, but I need to check if these changes doesn't break anything.

File: test/api_test.lua [hunk 1]
```diff
@@ -49,3 +50,21 @@ g.test_role_statistics = function()
...
+    t.assert_equals(result[utils.index.data], 'task_1')
+
+    cmd = ("return require('sharded_queue.api').statistics('%s')"):format(tube_name)
+    local result_m = g.queue_conn:eval(cmd)
+    local result_ro = g.queue_conn_1:eval(cmd)
+
+    t.assert_type(result_m, 'table')
+    t.assert_type(result_ro, 'table')
+    t.assert_equals(result_m, result_ro)
+end
```
Comment (vanyarock01) [inline test/api_test.lua:70]: Add a new line

File: sharded_queue/api.lua [hunk 2]
```diff
@@ -468,13 +468,10 @@ function sharded_queue.create_tube(tube_name, options)
...
 
 local function init(opts)
-    if opts.is_master then
-        rawset(_G, 'queue', sharded_queue)
-    end
+    rawset(_G, 'queue', sharded_queue)
 end
 
 local function apply_config(cfg, opts)
-    if opts.is_master then
```
Comment (olegrok) [inline sharded_queue/api.lua:477]: Since you remove `if` you need to reduce indent inside this section.

Comment (pnvasanth) [reply sharded_queue/api.lua:477]: done

File: test/api_test.lua (hunk 1 above)
Comment (pnvasanth) [reply test/api_test.lua]: done

File: test/api_test.lua [hunk 3]
```diff
@@ -49,3 +50,21 @@ g.test_role_statistics = function()
     local result = g.queue_conn:eval(cmd)
     t.assert_type(result, 'table')
 end
+
+g.test_role_statistics_read_only_router = function()
```
Comment (olegrok) [inline test/api_test.lua:54]: I suggest to add an assert that such instance is actually read_only.

Comment (pnvasanth) [reply test/api_test.lua:54]: done, its a good idea, thank you

**Review by akudiyar — APPROVED** [review]

**Pull request merged**